README.md

```diff
@@ -444,17 +444,17 @@ If `data_disks` is mentioned in your inventory, the following variables are requ
 * **size:** `required`. Specify the disk size expressed in GB. (es. `size: 1` means 1GB)
 * **pool:** `required`. Specify the pool where you want to store the additional disks.
 * **format:** `require`. Specify the filesystem format you want to apply to the disk. Available filesystems are specified below.
-* **mount_point:** `required`. Specify the mount point you want to create for the disk.
+* **mount_point:** `required`. Specify the mount point you want to create for the disk, use `none` if declaring a swap disk.
```

Comment:
Can you explain better the performance reasons why we introduce the swap disk? I think this could be useful for the others.